spec/requests/user_index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'UserIndix', type: :feature do
+  it 'response body includes correct placeholder text' do
+    visit users_index_path
+    expect(page).to have_content('List of all users')
```

Comment:
Also kindly test whether the correct template is being rendered by using eg., `expect(response).to render_template(:show)`